keel-ec2-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterSpec.kt

```diff
@@ -106,9 +108,9 @@ private fun ClusterSpec.resolveHealth(region: String): Health {
     cooldown = overrides[region]?.health?.cooldown ?: defaults.health?.cooldown ?: default.cooldown,
     warmup = overrides[region]?.health?.warmup ?: defaults.health?.warmup ?: default.warmup,
     healthCheckType = overrides[region]?.health?.healthCheckType ?: defaults.health?.healthCheckType
-      ?: default.healthCheckType,
+    ?: default.healthCheckType,
```

Comment:
nit: Original indentation looked right...

Reply:
I just do what `gradle spotlessApply` tells me to

Reply:
🤷‍♂️